com.unity.render-pipelines.universal/Documentation~/rendering/deferred-rendering-path.md

```diff
@@ -0,0 +1,422 @@
+
+Unity renders Materials with such shaders in the Forward Rendering Path. For the SSAO Renderer Feature to be able to calculate ambient occlusion for the Materials using the **Complex Lit** shader, Unity must render such Materials in the depth and normal prepass first. This is because Unity does not render those Materials in the G-buffer pass (GBufferPass). For more information, see [URP Pass tags](../urp-shaders/urp-shaderlab-pass-tags.md).
+
+#### General implementation notes
+
+For maximum platform compatibility, the URP Deferred Rendering Path uses the light stencil volume technique to perform the light accumulation.
+
+## Relevant code files
+
+This section contains the list of files that contain the code related to the Deferred Rendering Path.
+
+* The main class that handles the Deferred Rendering Path:
+
+    ```
+    com.unity.render-pipelines.universal\Runtime\DeferredLights.cs
+    ```
+
+* ScriptableRenderPass for the G-Buffer pass:
+
+    ```
+    com.unity.render-pipelines.universal\Runtime\Passes\GBufferPass.cs
+    ```
+
+* ScriptableRenderPass for the deferred shading pass:
+
+    ```
+    com.unity.render-pipelines.universal\Runtime\Passes\DeferredPass.cs
+    ```
+
+* Shader asset for the deferred shading:
+
+    ```
+    com.unity.render-pipelines.universal\Shaders\Utils\StencilDeferred.shader
+    ```
+
+* Utility functions for the deferred shading:
+
+    ```
+    com.unity.render-pipelines.universal\Shaders\Utils\Deferred.hlsl
+    ```
+
+* Utility functions for storing and loading the Material properties from the G-buffer:
+
+    ```
+    com.unity.render-pipelines.universal\Shaders\Utils\UnityGBuffer.hlsl
+    ```
+
+## ShaderLab Pass tags
+
+To enable Unity to render a shader in the Deferred Rendering Path, the shader must have a Pass with the following tag definition:
+
+`"LightMode" = "UniversalGBuffer"`
+
+Unity executes the shader with such `LightMode` tag during the G-buffer Pass.
+
+To indicate that Unity must render a certain Material in the Forward-only Pass in the Deferred Rendering Path, add the following tags to a shader Pass:
+
+`"LightMode" = "UniversalForwardOnly"`
+
+`"LightMode" = "DepthNormalsOnly"`
+
+To specify the shader lighting model (Lit, SimpleLit), use the `UniversalMaterialType` tag.
+
+For more information, see the section [URP Pass tags: LightMode](../urp-shaders/urp-shaderlab-pass-tags.md#lightmode).
+
+## Limitations and performance
+
+This section describes the limitations of the Deferred Rendering Path.
+
+### Terrain blending
+
+When blending more than four Terrain layers, the Deferred Rendering Path generates slightly different results from the Forward Rendering Path. This happens because in the Forward Rendering Path, Unity processes the first four layers separately from the next four layers when using multi-pass rendering.
```

Comment:
"from the next four layers using multi-pass rendering."

I would remove "when" because it always use multi-pass rendering to render the next four layers.